docs/api/index.rst

```diff
@@ -1,15 +1,87 @@
 .. _api-endpoints:
 
-The HTTP APIs
-#############
+API
+###
+
+Cheatsheet
+==========
+
++----------+----------------------------------------------------------------------------------------------+---------------------------------------------------------+
+| Method   | URI                                                                                          | Description                                             |
++==========+==============================================================================================+=========================================================+
+| `GET`    | :ref:`/ <api-utilities>`                                                                     | :ref:`Information about the running instance            |
+|          |                                                                                              | <api-utilities>`                                        |
++----------+----------------------------------------------------------------------------------------------+---------------------------------------------------------+
+| `POST`   | :ref:`/batch <batch>`                                                                        | :ref:`Send multiple operations in one request <batch>`  |
++----------+----------------------------------------------------------------------------------------------+---------------------------------------------------------+
+| `GET`    | :ref:`/__heartbeat__ <api-utilities>`                                                        | :ref:`Return the status of dependent services           |
+|          |                                                                                              | <api-utilities>`                                        |
++----------+----------------------------------------------------------------------------------------------+---------------------------------------------------------+
+| **Buckets**                                                                                                                                                       |
++----------+----------------------------------------------------------------------------------------------+---------------------------------------------------------+
+| `PUT`    | :ref:`/buckets/(bucket_id) <bucket-put>`                                                     | :ref:`Create or replaces a bucket <bucket-put>`         |
```

Comment:
`replace` (singular)

Either that, or `creates` (plural), in which case the voice of the other options below should be changed, too...